network/proxy.md

```diff
@@ -60,8 +60,8 @@ instead.
 
 | Variable      | Dockerfile example                                | `docker run` Example                                |
 |:--------------|:--------------------------------------------------|:----------------------------------------------------|
-| `HTTP_PROXY`  | `ENV HTTP_PROXY "http://127.0.0.1:3001"`          | `--env HTTP_PROXY="http://127.0.0.1:3001"`          |
-| `HTTPS_PROXY` | `ENV HTTPS_PROXY "https://127.0.0.1:3001"`        | `--env HTTPS_PROXY="https://127.0.0.1:3001"`        |
-| `FTP_PROXY`   | `ENV FTP_PROXY "ftp://127.0.0.1:3001"`            | `--env FTP_PROXY="ftp://127.0.0.1:3001"`            |
-| `NO_PROXY`    | `ENV NO_PROXY "*.test.example.com,.example2.com"` | `--env NO_PROXY="*.test.example.com,.example2.com"` |
+| `HTTP_PROXY`  | `ENV HTTP_PROXY "http://127.0.0.1:3001"`          | `--env http_proxy="http://127.0.0.1:3001"`          |
```

Comment:
Can you describe the problem you're having with Docker 18.03? Environment variables on Linux are case-sensitive, so both uppercase and lowercase can be used. FWIW, there is no official standard on the "right" capitalisation for these proxy env-vars (some tools on linux prefer lowercase, other uppercase, but mostly fall back to the other one if it's not set).

Using uppercase should work fine; in this example, I start a container having both the upper and lowercase version, and print the active environment variables inside the container (using the `env` linux command); both are set;

```bash
docker run --rm --env https_proxy="something" --env HTTPS_PROXY="something else" busybox env                                                                                                          PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HOSTNAME=112c56ef9f44
https_proxy=something
HTTPS_PROXY=something else
HOME=/root
```

Reply:
Interesting. My experience was that it did not set the environment variables at all when using uppercase letters. But I will have to double check this.

Reply:
@lindhe please let me know how you'd like to resolve this ticket. Thanks!

Reply:
Sorry, I forgot about it. Let's close it, and if I do get back to it eventually I'll reopen it.